exercises/concept/need-for-speed/.docs/introduction.md

```diff
@@ -1,18 +1,46 @@
 # Introduction
 
-In Go, a `struct` is a sequence of named elements called _fields_, each field having a name and type. The name of a field must be unique within the struct. `Structs` can be compared with the _class_ in the Object Oriented Programming paradigm.
+In Go, a `struct` is a sequence of named elements called _fields_, each field has a name and type. The name of a field must be unique within the struct.
+`Structs` can be compared with _classes_ in the Object-Oriented Programming paradigm.
 
-You create a new struct by using the `struct` keyword, a **_built-in type_**, and explicitly define the name and type of the fields as shown in the example below.
+You create a new struct by using the **_built-in type_** keywords `type` and `struct`, and explicitly define the name and type of the fields.
+For example, to define a `Person` struct:
 
 ```go
-type StructName struct{
-    field1 fieldType1
-    field2 fieldType2
+type Person struct{
+    name string
+    age int
 }
 ```
 
-Struct fields are accessed using a `.` notation.
+Once you have defined the `struct`, you need to create a new instance defining the fields _in order_, for example:
 
 ```go
-someStruct.someField = "someValue"
+person := Person{
+	"John",
+	20,
+}
```

Comment:
As this syntax is not really recommended as it creates brittle code (breaks when a field is added) I would recommend only showing it in the about.md of the struct concept.

Reply:
Sounds like a good idea. The exemplar solution also only uses the named field syntax.

Reply:
Sounds good, I'll take care of the exemplar solution too :)